tests/e2e/tests/pilot/ingress/controller_test.go

```diff
@@ -1,340 +0,0 @@
-// Copyright 2017 Istio Authors
```

Comment:
I think we still need this, ingress should still be supported.

Reply:
maybe just create an issue to add tests for ingress with the v2 implementation?